src/services/userService.js

```diff
@@ -0,0 +1,54 @@
+let Sequelize = require('sequelize');
```

Comment:
Static imports are always "const".
Consider fixing it everywhere in the codebase.

Reply:
resolved